common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -527,4 +528,9 @@ public class CommonValue {
                         + "\": {\"type\": \"long\"}\n"
                         + "    }\n"
                         + "}";
+        // Calculate Versions independently of OpenSearch core version
+        public static final Version VERSION_2_11_0 = Version.fromString("2.11.0");
```

Comment:
can we use `Version.V_2_11_0` directly from https://github.com/opensearch-project/OpenSearch/blob/main/libs/core/src/main/java/org/opensearch/Version.java#L98